user-guide/integrations/zapier-integration.md

```diff
@@ -0,0 +1,34 @@
+# Creating a Zap in Zapier for New Query Results
+
+You can connect Redash to Zapier and trigger an action every time a new line is added to you query results - for example, a Slack bot that announces every time a new order came in.
+The best way is to have a scheduled query that refreshes frequently so the notification you get is in real-time.
+
+For now you'll need to ask us for an invitation as it's still in beta - you can do that via Intercom (at the bottom right corner of every page in /help or inside the Redash webapp or [Slack](https://redash.slack.com/).
```

Comment:
Not sure what you got via email. I meant this one: https://zapier.com/developer/invite/32785/4910e4da7931a8f3a2124ebd85cc352b/